marklogic-client-api-functionaltests/src/test/java/com/marklogic/client/fastfunctest/TestOpticOnTriples.java

```diff
@@ -1423,4 +1423,136 @@ public void testPatternValuePermutations()
     assertEquals( "19", nodeValNeg1.path("PlayerAge").path("value").asText());
     assertEquals( "Midfielder", nodeValNeg1.path("PlayerPosition").path("value").asText());
   }
+
+	@Test
+	public void testShortestPathWithColumnInputs()
```

Comment:
Use `@ExtendWith(RequiresML12.class)` as an annotation on the method - that avoids the need for an "if" check. Also makes it easy to find all the tests that do require MarkLogic 12.